params.go

```diff
@@ -52,11 +53,18 @@ func (p *Params) ToCypherMap() string {
 		return "{}"
 	}
 
-	q := ""
+	// Create and sort a slice of keys to ensure cypher queries are created with consistent ordering
+	keys := make([]string, 0, len(p.params))
+	for k, _ := range p.params {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+
+	var sb strings.Builder
 
-	for _, v := range p.params {
-		q += fmt.Sprintf("%s,", v)
+	for _, k := range keys {
+		sb.WriteString(fmt.Sprintf("%s,", p.params[k]))
 	}
 
-	return "{" + strings.TrimSuffix(q, ",") + "}"
+	return fmt.Sprintf("{%s}", strings.TrimSuffix(sb.String(), ","))
```

Comment:
Nit: I think concatenation is still faster than `fmt.Sprintf` here.

Reply:
Updated this to use concatenation